test/e2e/framework/framework.go

```diff
@@ -73,20 +76,27 @@ const (
 	StartDockerUbuntu = "sudo systemctl start docker"
 	StopDockerUbuntu  = "sudo systemctl stop docker"
 
+	TMC            = "tmc"
+	TKG            = "tkg"
+	Kubernetes     = "kubernetes"
+	MissionControl = "mission-control"
 	TestDir        = ".tanzu-cli-e2e"
 	TestPluginsDir = ".e2e-test-plugins"
-	TargetTypeTMC  = "mission-control"
-	TargetTypeK8s  = "kubernetes"
+	TargetTypeTMC  = MissionControl
+	TargetTypeK8s  = Kubernetes
```

Comment:
Why are we defining these constants again when we can get those from `tanzu-plugin-runtime`?

Reply:
updated with runtime api